### ISSUE TITLE
New update: Hal Daumé III - new citations

### ISSUE THREAD

**Issue by postmillenial** [postmillenial/reading-list#1888]
From: scholaralerts-noreply@google.com<br>
Subject: Hal Daum&eacute; III - new citations<br>
First Attachment: http://ift.tt/19fNRe4<br>
Date: December 02, 2017 at 07:40AM<br>
<br>
<div style="font-family: arial,sans-serif; font-size: medium;">
<h3 style="font-weight: normal; margin: 0; font-size: medium;"><a href="http://ift.tt/2i8ShPN"><font size="3" color="#1A0DAB">Scalable Distributed Semantic Network for knowledge management in cyber physical system</font></a></h3>
<div><font size="2" color="#006621">S Song, Y Lin, B Guo, Q Di, R Lv&nbsp;- Journal of Parallel and Distributed Computing, 2017</font></div>
<div><font size="2" color="#222222">Abstract The remarkable growth of emerging technologies and computing paradigms in<br>
cyberspace and the cyber physical systems generate a huge mass of data sources. These<br>
different autonomous and heterogeneous data sources can contain complementary or<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDIUrL" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i8qoXZ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDIW2R" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i8qqz5"><font size="3" color="#1A0DAB">Multiple Instance Learning Networks for Fine-Grained Sentiment Analysis</font></a>
</h3>
<div><font size="2" color="#006621">S Angelidis, M Lapata&nbsp;- arXiv preprint arXiv:1711.09645, 2017</font></div>
<div><font size="2" color="#222222">Abstract: We consider the task of fine-grained sentiment analysis from the perspective of<br>
multiple instance learning (MIL). Our neural model is trained on document sentiment labels,<br>
and learns to predict the sentiment of text segments, ie sentences or elementary discourse<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDIX6V" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i8qsab" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDrBac" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2icuWNf"><font size="3" color="#1A0DAB">Phrase Table Induction Using In-Domain Monolingual Data for Domain Adaptation in Statistical Machine Translation</font></a>
</h3>
<div><font size="2" color="#006621">B Marie, A Fujita&nbsp;- Transactions of the Association for Computational&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract We present a new framework to induce an in-domain phrase table from in-domain<br>
monolingual data that can be used to adapt a general-domain statistical machine translation<br>
system to the targeted domain. Our method first compiles sets of phrases in source and<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDJ0Q9" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2icx6fK" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDJ1Ud" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i8quPl"><font size="3" color="#1A0DAB">Modelling Domain Relationships for Transfer Learning on Retrieval-based Question Answering Systems in E-commerce</font></a>
</h3>
<div><font size="2" color="#006621">J Yu, M Qiu, J Jiang, J Huang, S Song, W Chu, H Chen&nbsp;- arXiv preprint arXiv&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract: In this paper, we study transfer learning for the PI and NLI problems, aiming to<br>
propose a general framework, which can effectively and efficiently adapt the shared<br>
knowledge learned from a resource-rich source domain to a resource-poor target domain.<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jEm9ni" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i8qvTp" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jExPGC" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2nmrrsS"><font size="3" color="#1A0DAB">Improving the Accuracy of Pre-trained Word Embeddings for Sentiment Analysis</font></a>
</h3>
<div><font size="2" color="#006621">SM Rezaeinia, A Ghodsi, R Rahmani&nbsp;- arXiv preprint arXiv:1711.08609, 2017</font></div>
<div><font size="2" color="#222222">Abstract: Sentiment analysis is one of the well-known tasks and fast growing research areas<br>
in natural language processing (NLP) and text classifications. This technique has become<br>
an essential part of a wide range of applications including politics, business, advertising and<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2zGv7HI" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2nnWqVh" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2zJQSq1" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i8qy1x"><font size="3" color="#1A0DAB">Convolutional Image Captioning</font></a>
</h3>
<div><font size="2" color="#006621">J Aneja, A Deshpande, A Schwing&nbsp;- arXiv preprint arXiv:1711.09151, 2017</font></div>
<div><font size="2" color="#222222">Abstract: Image captioning is an important but challenging task, applicable to virtual<br>
assistants, editing tools, image indexing, and support of the disabled. Its challenges are due<br>
to the variability and ambiguity of possible image descriptions. In recent years significant<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jECGb1" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i8qz5B" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDJ7Lz" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[HTML]</font> <a href="http://ift.tt/2i8qzCD"><font size="3" color="#1A0DAB">Using deep learning and Google Street View to estimate the demographic makeup of neighborhoods across the United States</font></a>
</h3>
<div><font size="2" color="#006621">T Gebru, J Krause, Y Wang, D Chen, J Deng, EL Aiden&hellip;&nbsp;- Proceedings of the National&nbsp;&hellip;, 2017</font></div>
<div><font size="2" color="#222222">Abstract The United States spends more than $250 million each year on the American<br>
Community Survey (ACS), a labor-intensive door-to-door study that measures statistics<br>
relating to race, gender, education, occupation, unemployment, and other demographic<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDbGbQ" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i8qBdJ" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDJ9mF" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<h3 style="font-weight: normal; margin: 0; font-size: medium;">
<font size="2" color="#1A0DAB">[PDF]</font> <a href="http://ift.tt/2i7XO9g"><font size="3" color="#1A0DAB">Large-scale Machine Learning over Graphs</font></a>
</h3>
<div><font size="2" color="#006621">H Liu - 2017</font></div>
<div><font size="2" color="#222222">Abstract Graphs are ubiquitous in statistical modeling and a broad range of machine<br>
learning applications. Examples are social networks, natural language dependency<br>
structures, latent interrelationships among tasks, and neural network topologies. Despite<br></font></div>
<table>
<tbody>
<tr>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDJaqJ" style="text-decoration: none;"><img alt="Google+" src="http://ift.tt/2c8KCRB" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2i8qCOP" style="text-decoration: none;"><img alt="Facebook" src="http://ift.tt/2c8JI7K" border="0" height="16" width="16"></a></td>
<td width="16" style="padding-right: 6px;"><a href="http://ift.tt/2jDbHMW" style="text-decoration: none;"><img alt="Twitter" src="http://ift.tt/2c8Je1i" border="0" height="16" width="16"></a></td>
</tr>
</tbody>
</table>
<br>
<hr noshade size="1">
<font size="2" color="#222222">This Google Scholar Alert is brought to you by Google.<br>
<br>
<a href="http://ift.tt/2vUKjPi"><font color="#1A0DAB">Cancel alert</font></a><br>
<a href="http://ift.tt/2eWhNpX"><font color="#1A0DAB">List my alerts</font></a></font>
</div>
<br>
<br>
via Gmail
